var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -411,5 +413,14 @@ def cmake_args(self):
 
     @run_after('install')
     def post_install(self):
+        if '+clang' in self.spec and '+python' in self.spec:
+            python_version = self.spec['python'].version.up_to(2)
+            site_packages_dir = join_path(
```

Comment:
Is `site_packages_dir` not already in module scope via the `python` package's `setup_dependent_package()` method?  See [python/package.py:543](https://github.com/LLNL/spack/blob/develop/var/spack/repos/builtin/packages/python/package.py#L543). I think you can ski making this path yourself.

Reply:
Ah, I didn't know. Thanks :) → Updated patch.